lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -111,7 +117,14 @@
                                 url: 'my-teams',
                                 view: this.myTeamsView
                             }, {
-                                title: gettext('Browse'),
+                                // Translators: This will be shown
+                                // to users as "Browse teams" if
+                                // they are using a screenreader,
+                                // and "Browse" otherwise.
+                                title: _.escape(gettext('Browse')) +
+                                    '<span class="sr"> ' +
+                                    gettext('teams') +
```

Comment:
@peter-fogg I'm not sure this will work well for internationalization. Consider languages where the word for "Browse" comes after the word for "teams". Might be better to do something like:
```js
interpolate(gettext("Browse %(sr_start)s teams %(sr_end)s"),
  {"sr_start": '<span class="sr">', "sr_end": '</span>'},
  true
)
```
Just be sure to provide a translators comment that explains what "sr_start" and "sr_end" mean to the translators. Also, I believe that translators comments can't span multiple lines, due to limitations in our internationalization tools.